zcash_client_sqlite/src/wallet/commitment_tree.rs

```diff
@@ -206,13 +206,6 @@ impl<'conn, 'a: 'conn, H: HashSer, const SHARD_HEIGHT: u8> ShardStore
         update_checkpoint_with(self.conn, self.table_prefix, *checkpoint_id, update)
     }
 
-    fn for_each_checkpoint<F>(&self, limit: usize, callback: F) -> Result<(), Self::Error>
```

Comment:
I'm not sure I understand what this change is for. It would be better to avoid modifying the sqlite client crate if possible

Reply:
This was added by mistake in a commit thats already in main. Look like 20 lines before and you'll see that this is a duplicate definition.